tests/rbac_test.go

```diff
@@ -0,0 +1,227 @@
+package tests_test
+
+import (
+	"flag"
+	"fmt"
+	"time"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kubevirt.io/kubevirt-ansible/tests"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	ktests "kubevirt.io/kubevirt/tests"
+)
+
+var _ = Describe("RBAC", func() {
+	flag.Parse()
+
+	var vmi tests.VMManifest
+
+	const NamespaceTestRandom  = "kubevirt-test-random"
+
+	vmi.Name = "vm-cirros"
+	vmi.Manifest = "tests/manifests/vm-cirros.yaml"
+
+	virtClient, err := kubecli.GetKubevirtClient()
+	ktests.PanicOnError(err)
+
+	BeforeEach(func() {
+		By(fmt.Sprintf("Login with the user %s", tests.UsernameAdminUser))
+		args := []string{"login", "-u", tests.UsernameAdminUser, "-p", "123456"}
+		_, err := ktests.RunCommand("oc", args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		ktests.BeforeTestCleanup()
+		tests.CreateUser(tests.UsernameTestUser)
+	})
+
+	AfterEach(func() {
+		By(fmt.Sprintf("Login with the user %s", tests.UsernameAdminUser))
+		args := []string{"login", "-u", tests.UsernameAdminUser, "-p", "123456"}
+		_, err := ktests.RunCommand("oc", args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		tests.DeleteUser(tests.UsernameTestUser)
+
+		By(fmt.Sprintf("Delete identity of user %s", tests.UsernameTestUser))
+		args = []string{"delete", "identity", "allow_all_auth:"+tests.UsernameTestUser}
+		_, err = ktests.RunCommand("oc", args...)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	Describe("RBAC with default permission",func ()  {
+		It("should allow to access subresource endpoint", func ()  {
+			By(fmt.Sprintf("Login with the user %s", tests.UsernameTestUser))
+			args := []string{"login", "-u", tests.UsernameTestUser, "-p", "123456"}
+			_, err := ktests.RunCommand("oc", args...)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Create a namespace")
+			args = []string{"new-project", NamespaceTestRandom}
+			_, err = ktests.RunCommand("oc", args...)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Create a VM with yaml.")
+			args = []string{"create", "-f", vmi.Manifest}
+			_, err = ktests.RunCommand("oc", args...)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Check VMI should create.")
+			args = []string{"start", vmi.Name}
+			_, err = ktests.RunCommand("virtctl", args...)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Check the vmi is running.")
+			args = []string{"get", "vmi", vmi.Name, "--template", "{{.status.phase}}"}
+			Eventually(func() string {
+				output, err := ktests.RunCommand("oc", args...)
+				Expect(err).ToNot(HaveOccurred())
+				return output
+			}, time.Minute*10).Should(Equal("Running"))
+
+			By("Connect via console.")
+			vm, err := virtClient.VirtualMachineInstance(NamespaceTestRandom).Get(vmi.Name, &metav1.GetOptions{})
+			expecter, err := ktests.LoggedInCirrosExpecter(vm)
+			Expect(err).ToNot(HaveOccurred())
+			defer expecter.Close()
```

Comment:
We don't really want to keep the `expecter` open for the whole `It` closure.
```suggestion
			expecter.Close()
```